src/ontology/fbbt-edit.obo

```diff
@@ -123372,6 +123418,30 @@ relationship: part_of FBbt:00100311 ! cortical zone of lymph gland primary lobe
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-05-27T11:11:09Z
 
+[Term]
+id: FBbt:00058153
+name: high salt-sensing neuron of the labellum
+def: "A high salt-sensing neuron that innervates a short or long sensillum of the labellum (Montell, 2009)." [FlyBase:FBrf0208984]
```

Comment:
These are a subset of the labellar bitter-sensing neurons - the bitter neurons in S-a and S-b sensilla. From FBrf0229608:
>Bitter taste neurons in S-type sensilla also respond to high concentrations of salt